source/gui/settingsDialogs.py

```diff
@@ -1111,12 +1111,27 @@ def makeSettings(self, settingsSizer):
 		self.lineNumberCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report line &numbers"))
 		self.lineNumberCheckBox.SetValue(config.conf["documentFormatting"]["reportLineNumber"])
 		settingsSizer.Add(self.lineNumberCheckBox,border=10,flag=wx.BOTTOM)
-		# Translators: This message is presented in the document formatting settings dialogue
-		# If this option is selected, NVDA will cound the leading spaces and tabs of a line and speak it.
-		#
-		self.lineIndentationCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report l&ine indentation"))
-		self.lineIndentationCheckBox.SetValue(config.conf["documentFormatting"]["reportLineIndentation"])
-		settingsSizer.Add(self.lineIndentationCheckBox,border=10,flag=wx.BOTTOM)
+		sizer=wx.BoxSizer(wx.HORIZONTAL)
+		# Translators: This is the label for a combobox in the
+		# Document  Formatting  dialog (possible choices are Off, Speech, Tones, or Both.

```

Comment:
nit: The translator comment specifies the choices, but doesn't specify what they're for. I realise they can figure this out from the text. Still, I think it'd be clearer to say something like:

> This is the label for a combobox in the Document Formatting dialog wherein the user can choose how line indentation is reported (off, speech, tones or both).